### PR TITLE
tidb-in-kubernetes: local-pv-management

### PR DESCRIPTION
**Pull request by cofyc**
https://github.com/cofyc/docs-cn/blob/local-pv-management/tidb-in-kubernetes/manage-local-pv.md

### CODE REVIEW

File: tidb-in-kubernetes/local-pv-management.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# 本地 PV 管理
+
+TiDB 是高可用数据库，存储层 TiKV 会对数据进行复制，容忍节点不可用。TiKV 使用高
+IOPS、吞吐量的本地存储，比如 Local SSDs 时，可提高数据库性能。
+
+Kubernetes 当前支持静态分配的本地存储。可使用
+[local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) 项目中的
+local-volume-provisioner 程序创建本地存储对象。流程如下：
+
+- 参考[操作文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md)在集群节点预分配本地存储
```
Comment (TomShawn) [inline tidb-in-kubernetes/local-pv-management.md]: 1. 参考 Kubernetes 提供的 [操作文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md)，在 TiKV 集群节点中预分配本地存储。

File: tidb-in-kubernetes/local-pv-management.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- 参考[部署例子](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm)部署 local-volume-provisioner 程序
```
Comment (TomShawn) [inline tidb-in-kubernetes/local-pv-management.md]: 2. 参考 Helm 的 [部署案例](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm)，部署 local-volume-provisioner 程序。

File: tidb-in-kubernetes/local-pv-management.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+更多可了解 [Kubernetes 本地存储](https://kubernetes.io/docs/concepts/storage/volumes/#local) 和 [local-static-provisioner 文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview)。
```
Comment (TomShawn) [inline tidb-in-kubernetes/local-pv-management.md]: 更多信息，可参阅 [Kubernetes 本地存储](https://kubernetes.io/docs/concepts/storage/volumes/#local) 和 [local-static-provisioner 文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview)。

**Comment by cofyc** [comment]
Thanks for reviewing this! Updated.

**Comment by TomShawn** [comment]
LGTM !

File: tidb-in-kubernetes/local-pv-management.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]: Generally, for a how-to document that users can follow it to do some operations, we use a tile or file name that is composed of verbs if possible. This is the optimized naming rule for SEO. So I suggest changing this filename to `manage-local-pv.md`.

File: tidb-in-kubernetes/local-pv-management.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+title: 本地 PV 管理
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]:
```suggestion
title: 管理本地 PV
```

File: tidb-in-kubernetes/local-pv-management.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+category: how-to
+---
+
+# 本地 PV 管理
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]:
```suggestion
# 管理本地 PV
```

File: tidb-in-kubernetes/local-pv-management.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+TiDB 是一款高可用的数据库。TiDB 的存储层 TiKV 对数据进行复制，容忍节点不可用。TiKV 使用高
+ IOPS 和高吞吐量的本地存储，比如 Local SSDs 时，可提高数据库性能。
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]: Please combine this line the previous one. DO NOT manually break a line. Thanks!

File: tidb-in-kubernetes/local-pv-management.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+Kubernetes 当前支持静态分配的本地存储。可使用
+[local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) 项目中的
+local-volume-provisioner 程序创建本地存储对象。创建流程如下：
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]: ditto for this para.

File: tidb-in-kubernetes/local-pv-management.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+ 1. 参考 Kubernetes 提供的 [操作文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md)，在 TiKV 集群节点中预分配本地存储。
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]:
```suggestion
1. 参考 Kubernetes 提供的[操作文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md)，在 TiKV 集群节点中预分配本地存储。
```

File: tidb-in-kubernetes/local-pv-management.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+2. 参考 Helm 的 [部署案例](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm)，部署 local-volume-provisioner 程序。
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]:
```suggestion
2. 参考 Helm 的[部署案例](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm)，部署 local-volume-provisioner 程序。
```

File: tidb-in-kubernetes/local-pv-management.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+更多信息，可参阅 [Kubernetes 本地存储](https://kubernetes.io/docs/concepts/storage/volumes/#local) 和 [local-static-provisioner 文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview)。
```
Comment (lilin90) [inline tidb-in-kubernetes/local-pv-management.md]:
```suggestion
更多信息，可参阅 [Kubernetes 本地存储](https://kubernetes.io/docs/concepts/storage/volumes/#local)和 [local-static-provisioner 文档](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview)。
```

File: tidb-in-kubernetes/local-pv-management.md (hunk 4 above)
Comment (cofyc) [reply tidb-in-kubernetes/local-pv-management.md]: Got it!

**Comment by cofyc** [comment]
All comments are addressed. PTAL again.

**Comment by lilin90** [comment]
@weekface @aylei PTAL~ Thanks!

**Pull request merged**